modules/ROOT/partials/configuration/exportword.adoc

```diff
@@ -38,12 +38,36 @@ tinymce.init({
   toolbar: 'exportword',
   exportword_service_url: '<service_URL>',
   exportword_converter_options: {
-    header: [
-      {
-        html: '<h1>First page header.</h1>', //example
-        css: 'h1 { font-size: 30px; }', //example
+    headers: {
+      first: {
+        html: "<p>My document header</p>",
+        css: "p { color: gray; }"
+      },
+      default: {
+        html: "<p>My default header</p>",
+        css: "p { color: blue; }"
       }
-    ],
+    },
+    footers: {
+      odd: {
+        html: "<p>My odd footer</p>",
+        css: "p { font-size: 10px; }"
+      },
+      even: {
+        html: "<p>My even footer</p>",
+        css: "p { font-size: 12px; }"
+      }
+    },
+    document: {
+      orientation: "landscape",
+      size: "a3",
```

Comment:
- [ ] Wait until it is determined whether lowercase document size names are supported or not.